demo/src/main/java/com/example/demo/dto/client/CreateClientDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.example.demo.dto.client;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@RequiredArgsConstructor
+@Getter
+@Setter
+public class CreateClientDTO {
+    private Long id;
```

Comment:
1. Don't transfer id.
2. We need to validate not null fields (read about validation). For example @NotNull, @NotEmpty...
3. No public fields in DTO